modules/ROOT/pages/as2/as2-connectorMule4.adoc

```diff
@@ -31,10 +35,10 @@ You should have familiarity with AS2.
 
 The AS2 version 5 connector requires the `Async MDN` to be handled as a separate flow. The retry of AS2 Message Send is not automatic. The application has to handle the retry upon failure in an MDN or upon failure to receive an MDN.
```

Comment:
This paragraph that starts on line 36 now is confusing. I'm not sure what it is saying.  Maybe:
AS2 Connector 5.0.0 requires `Async MDN` to be handled as a separate flow, as AS2 Message Send retry is not automatic. The application must handle retry upon failure in an MDN, or upon failure to receive an MDN.

Reply:
I really don't know what it's saying though. An MDN is like a return receipt. So how is retry upon failure handled in an MDN?

Reply:
These should be bullets. Fixed.